Cargo.toml

```diff
@@ -12,3 +12,6 @@ libc = "0.2"
 # rust-htslib = { version = "0.26", default-features = false }
 rust-htslib = { git = "https://github.com/rust-bio/rust-htslib.git", rev = "e36c954f5b7830a95d987f0a6b45b62d4424d62b", default-features = false }
 star-sys = { version = "0.2", path = "star-sys" }
+
+[profile.release]
+debug = 1
```

Comment:
@nlhepler Was this line intentional?

Reply:
Yes, actually. Though I did forget to call it out in the PR notes.

We use debug=1 elsewhere to capture useful stack traces from release code: https://github.com/10XDev/cellranger/blob/master/lib/rust/Cargo.toml#L28-L29. 

It is my understanding, though, that the final linking binary has ultimate control over this.